.gitmodules

```diff
@@ -1,3 +1,3 @@
 [submodule "imported/llvm-dialects"]
 	path = imported/llvm-dialects
-	url = ../llvm-dialects
+	url = git@github.com:GPUOpen-Drivers/llvm-dialects.git
```

Comment:
This looks unintended?

Reply:
Yes, apologies didn't mean to add that file, should be fixed in latest commit